syntax/blocks.js

```diff
@@ -54,10 +52,8 @@ export function parseInputNumber(part) {
 
 // used for procDefs
 export function parseSpec(spec) {
-  const parts = spec.split(splitPat).filter(x => !!x)
-  const inputs = parts.filter(p => {
-    return inputPat.test(p)
-  })
+  const parts = spec.split(splitPat).filter(x => x)
```

Comment:
Do we not want to keep `!!` to force the bool coercion? Or is there a more specific test we can do here, like the `x != null` from above?

Reply:
I don't think boolean coercion is necessary.